packages/hop-node/src/theGraph/getUnsetTransferRoots.ts

```diff
@@ -0,0 +1,40 @@
+import getTransferRootBonded from 'src/theGraph/getTransferRootBonded'
+import getTransferRootConfirmed from 'src/theGraph/getTransferRootConfirmed'
+import getTransferRootSet from 'src/theGraph/getTransferRootSet'
+import { Chain } from 'src/constants'
+import { getAllChains } from 'src/config'
+
+export default async function getUnsetTransferRoots (startDate: number, endDate: number) {
+  // An empty token address represents all tokens.
+  const token = ''
```

Comment:
If the `token` parameter is optional then it should be set as optional and as the last parameter since it's not clear in the function signature that an empty string should be used